public/app/features/dashboard/components/SaveDashboard/forms/SaveDashboardAsForm.test.tsx

```diff
@@ -50,8 +50,8 @@ const renderAndSubmitForm = async (
 
   // @ts-ignore strict null error below
```

Comment:
Can you check if this `@ts-ignore` comment is still required? Seems like it wouldn't be?

Reply:
Yep not needed will remove 👍  (also on the other 'enzyme -> RTL' branch)